scripts/golangci-lint.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# prints error if a specific command does not exist
+function panic_if_command_not_exists() {
+  if ! command -v "$1" >>/dev/null 2>/dev/null; then
+    echo -e "\033[0;33m$1\033[0m command is not available - please install it first"
+    exit 1
+  fi
+}
+
+# installs the golangci-lint of the specific version to the specific directory
+function install_golangci_lint() {
+  local version="$1"
+
+  # check presence and version of golangci-lint
+  if [[ -f third_party/golangci-lint ]]; then
```

Comment:
Do we need it? this script should be only called using Makefile (`make lint`) which is in the root folder of the project.
I don't want engineers to directly call this code, since they need to provide the golangci-lint version manually for this function (and also read this script), however `make lint` relies on [this](https://github.com/VolumeFi/pigeon/blob/e28c993e7a0c274201c9ccc0d03d905c2725b77a/Makefile#L3) variable as version.

Reply:
I will often call specific commands out of a makefile when debugging them :-) I would prefer we had this if the script super duper depends on the path. Another method you can use is something like `cd "$(git rev-parse --show-toplevel)"`, which puts us in the root git directory. But I'd prefer one or the other. Also in this case for the original you probably want `cd "$(dirname "$0")/.."`

Reply:
Got you, will do that then

Reply:
👍

Reply:
I'd prefer `cd "$(dirname "$0")/.."` in case if we run the script from outside the project (so outside from git repo) then.